src/components/app/MobileMainHeader.jsx

```
@@ -1,28 +1,34 @@
+import ROUTER_PATHS from '../../routes/RouterPath';
+import { useState, useRef, useEffect } from 'react';
+import MobileSidebar from './MobileSidebar';
+import useAuthStore from '../../store/login/useAuthStore';
 
 const MobileMainHeader = () => {
-  const navigate = useNavigate()
-  const [isSidebarOpen, setIsSidebarOpen] = useState(false)
+  const navigate = useNavigate();
+  const [isSidebarOpen, setIsSidebarOpen] = useState(false);
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const [selectedOption, setSelectedOption] = useState('default');
```

ESLint

[error] 'selectedOption' is assigned a value but never used.

(no-unused-vars)